modules/vsphere-problem-detector-about.adoc

```diff
@@ -14,7 +14,7 @@ The Operator runs in the `openshift-cluster-storage-operator` namespace and is s
 
 The Operator runs the checks according to the following schedule:
 
-* The checks run every 8 hours.
+* The checks run every 1 hour.
```

Comment:
```suggestion
* The checks run every hour.
```

Reply:
Makes sense! and now my changes don't sound right to me. Thanks for the catch of the obvious! :smile: